asv/graph.py

```diff
@@ -118,13 +118,15 @@ def resample_data(self, val):
 
         new_val = []
         j = 0
-        for i in xrange(min_time + step_size, max_time + step_size, step_size):
+        i = min_time + step_size
```

Comment:
Javascript timestamps handled here overflow the 32-bit integers on
win32, and xrange on Py2 only works with native size integers, so using
it is not possible here.